content/docs/getting-started/adding-a-catalog-item/cli/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Roadie CLI
+publishedDate: '2022-04-04T14:00:00.0Z'
+description: How to configure the Roadie CLI to create a catalog item.
+
+humanName: Roadie CLI
+examples:
+  - name: component
```

Comment:
I would add name as it is easier to retrieve it later

Reply:
This needs to be up cased on the page: i.e. Component